src/EnergyPlus/AirflowNetwork/include/AirflowNetwork/Solver.hpp

```diff
@@ -260,9 +260,9 @@ namespace AirflowNetwork {
                     int const FLAG               // mode of operation
         );
 
-        void manage_balance(Optional_bool_const FirstHVACIteration = _, // True when solution technique on first iteration
-                            Optional_int_const Iter = _,                // Iteration number
-                            Optional_bool ResimulateAirZone = _         // True when solution technique on third iteration
+        void manage_balance(ObjexxFCL::Optional_bool_const FirstHVACIteration = _, // True when solution technique on first iteration
```

Comment:
The vast majority of the changes in this PR are adding an explicit namespace to all the Optional objects. The main purpose for doing that is to make it easier to find all the objects without having to deal with false positives from comments and variable names containing "Optional". I did consider giving Omit (`_`) the same treatment, but in the end decided that would be taunting `clang-format` a second time.